games-engines/odamex/odamex-0.8.1.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+WX_GTK_VER="3.0"
+inherit cmake-utils desktop wxwidgets xdg
+
+DESCRIPTION="Online multiplayer, free software engine for Doom and Doom II"
+HOMEPAGE="https://odamex.net/"
+SRC_URI="mirror://sourceforge/${PN}/Odamex/${PV}/${PN}-src-${PV}.tar.bz2"
+
+LICENSE="GPL-2+ MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client master upnp +odalaunch portmidi server"
+REQUIRED_USE="
+	|| ( client master server )
+	portmidi? ( client )
+	upnp? ( server )"
```

Comment:
Unless it causes problems with the build, I think including `portmidi` and `upnp` here is overkill. No one is going to disable the client but wonder why the server has no MIDI support. It is more likely to annoy people who have these flags set globally.